packages/side-nav/src/SideNav.spec.tsx

```diff
@@ -63,10 +63,7 @@ describe('packages/side-nav', () => {
       });
 
       test('it sets the correct role for the item', () => {
-        expect(renderedEls.itemEl?.parentNode).toHaveAttribute(
-          'role',
-          'menuitem',
-        );
+        expect(renderedEls.itemEl).toHaveAttribute('role', 'menuitem');
```

Comment:
I would prefer this block:
```
test('it sets the correct role for the item and its container', () => {
   expect(renderedEls.itemEl).toHaveAttribute('role', 'menuitem');
   expect(renderedEls.itemEl?.parentNode).toHaveAttribute('role', 'none');
});
```

Just so the test coverage maintains this in the future, otherwise I could see myself or someone else modifying this without thinking once we've all stopped thinking so much about IA Navigation.

Reply:
Great, will do.